app/v2/generators/DepartureMessageGenerator.scala

```diff
@@ -42,7 +42,7 @@ import java.time.Clock
 @ImplementedBy(classOf[DepartureMessageGeneratorImpl])
 trait DepartureMessageGenerator extends MessageGenerator
 
-class DepartureMessageGeneratorImpl @Inject()(clock: Clock) extends Generators with DepartureMessageGenerator {
+class DepartureMessageGeneratorImpl @Inject() (clock: Clock) extends Generators with DepartureMessageGenerator {
```

Comment:
Build keep this as previous. As build got failed due to this formatting looks like